fixup-test-helper

From: Bryan Larsen <bryan@larsen.st>

The generators did not replace our old fixtures based
`test_helper.rb`, so lets fix that up:

SHOW_PATCH
---
 test/test_helper.rb |    6 +-----
 1 files changed, 1 insertions(+), 5 deletions(-)

diff --git a/test/test_helper.rb b/test/test_helper.rb
index 8bf1192..6f13d1f 100644
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -3,11 +3,7 @@ require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 
 class ActiveSupport::TestCase
-  # Setup all fixtures in test/fixtures/*.(yml|csv) for all tests in alphabetical order.
-  #
-  # Note: You'll currently still have to declare fixtures explicitly in integration tests
-  # -- they do not yet inherit this setting
-  fixtures :all
+  include Factory::Syntax::Methods
 
   # Add more helper methods to be used by all tests here...
 end
